activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -163,7 +163,18 @@ def migrations_paths # :nodoc:
       end
 
       def migration_context # :nodoc:
-        MigrationContext.new(migrations_paths)
+        MigrationContext.new(migrations_paths, schema_migration)
+      end
+
+      def schema_migration
```

Comment:
Is this intended to be exposed in the api doc?

Reply:
If the class was not, I think we shouldn't make the method public.

Reply:
Fixed